libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp

```diff
@@ -144,28 +144,29 @@ bool AP_InertialSensor::BatchSampler::Write_ISBD() const
 // @Description: Filter Tuning Message - per motor
 // @Field: TimeUS: microseconds since system startup
 // @Field: I: instance
-// @Field: NDn: number of active dynamic harmonic notches
-// @Field: NF1: dynamic harmonic notch centre frequency for motor 1
-// @Field: NF2: dynamic harmonic notch centre frequency for motor 2
-// @Field: NF3: dynamic harmonic notch centre frequency for motor 3
-// @Field: NF4: dynamic harmonic notch centre frequency for motor 4
-// @Field: NF5: dynamic harmonic notch centre frequency for motor 5
-// @Field: NF6: dynamic harmonic notch centre frequency for motor 6
-// @Field: NF7: dynamic harmonic notch centre frequency for motor 7
-// @Field: NF8: dynamic harmonic notch centre frequency for motor 8
-// @Field: NF9: dynamic harmonic notch centre frequency for motor 9
-// @Field: NF10: dynamic harmonic notch centre frequency for motor 10
-// @Field: NF11: dynamic harmonic notch centre frequency for motor 11
-// @Field: NF12: dynamic harmonic notch centre frequency for motor 12
+// @Field: NDn: number of active harmonic notches
+// @Field: NF1: desired notch centre frequency for motor 1
```

Comment:
I would prefer we kept the word "harmonic" in here. So "desired harmonic notch centre frequency for motor 1", doesn't cost any flash and a little clearer I think

Reply:
changed